redhat/ubi8-openjdk-17.yaml

```diff
@@ -3,7 +3,8 @@ osbs:
     container:
       compose:
         pulp_repos: true
-        packages: []
+        packages:
+        - java-17-openjdk-headless
```

Comment:
AFAIK, for the compose to work it needs to list all installed packages. For `ubi8/openjdk-17` this should be `java-17-openjdk{,-devel,-headless}`.

Reply:
Ah yes I believe you are right, good catch!